workshop/ReadMe

```diff
@@ -0,0 +1,2 @@
+Commands to run
+docker build -t automation .  && docker run automation
```

Comment:
How do you specify whether to build the `TestBuild` or the `ProdBuild` ?

Reply:
By using -- target TestBuild.

docker build --target TestBuild -t automation .